crates/bevy_ui/src/widget/text.rs

```diff
@@ -34,8 +33,8 @@ pub fn text_constraint(min_size: Val, size: Val, max_size: Val, scale_factor: f6
     }
 }
 
-/// Computes the size of a text block and updates the TextGlyphs with the
-/// new computed glyphs from the layout
+/// Computes the size of a text block and updates the `TextGlyphs` with the new computed glyphs
```

Comment:
We don't have a type named `TextGlyphs` (or `TextGlyph`). Not sure if this documentation is outdated?

Reply:
Yeah, this is outdated. We can fix that later though.